contents/handbook/team-structure.md

```diff
@@ -1,35 +1,108 @@
+- [ ] Team lead adds/removes from relevant [GitHub teams](https://github.com/orgs/PostHog/teams).
+- [ ] Team lead adds/removes from Slack `@` groups / team mentions (e.g. `@core-experience`).
+- [ ] Team lead creates a new [teams page and adds content](/teams), team members to it
+- [ ] Team lead creates 1:1s with new team members.
+- [ ] Team lead creates stand-ups, planning sessions, and any other team rituals, including sprint templates.
+- [ ] Team lead creates a new team Slack channel, with descriptions and group handle
+- [ ] Team lead opens a PR on `posthog.com` documenting the change in the handbook.
 
-We’re still an early stage company, and so is our product. We strongly value moving and shipping fast (see [core values](/handbook/company/values)) and we constantly iterate not only our product but our organization too. As this happens, it may be quite possible that organizational changes occur. This mostly happens at the [small teams](/handbook/company/small-teams) level. To make the changes smoother for everyone, here’s the checklist we use:
+## I've created my new small team. Now what?
 
-- [ ] Discuss with relevant team leads (and/or managers if applicable).
-- [ ] Discuss with relevant team member(s).
-- [ ] Open a PR on `posthog.com` documenting the change on the handbook.
-    - [ ] Update the relevant Small Team page.
-    - [ ] Update the [Org Chart](https://app.deel.com/organization-chart/organization/834ac289-7c04-4d93-91f0-8922c5664b77?groupBy=group-by-report) in Deel.
-- [ ] Let everyone else in the company know in the next all hands session.
-- [ ] Add/remove from relevant [Sentry teams](https://sentry.io/settings/posthog/teams/).
-- [ ] Add/remove from relevant [GitHub teams](https://github.com/orgs/PostHog/teams).
-- [ ] Add/remove from Slack `@` groups / team mentions (e.g. `@core-experience`).
+Congrats! You've been nominated as the lead for a new or existing product. We've launched a number of products at PostHog over the years, and this is our mini-handbook for how to do it well.
 
-### Team Lead checklist
-- [ ] Onboarding 1:1 with new team member.
-- [ ] Add to stand-ups, planning sessions, and any other team rituals.
-- [ ] Update Slack channel description, and invite to Slack channel if needed.
+> If you're a new lead for an existing product, not all of this doc will apply to you. However, it's still worth a read, as it describes the launch process and how we got to where we are.
+
+### Step 1: Plan your product
+
+Even if you have a general idea of what your product will do, the RFC process will help you think through the ins-and-outs of your product, collect feedback, and start thinking about launch.
+
+Use the [New product RFC template](https://github.com/PostHog/product-internal/blob/main/requests-for-comments/templates/request-for-comments-new-product.md) as a guide. You don't have to follow it precisely - feel free to include whatever other sections you think are relevant. 
```

Comment:
All this stuff kinda seems like it doesn't really fit inside this doc? The doc is called "team structure" but this is "team _process_" and it's only for engineering teams. 

I'd personally go back to the way it was before, with this doc in the product or engineering area.

Also I didn't see a PR for the issue template change but I think there should be an actual checklist item to read this doc if you're an engineering / product team. It's a task they should do.